.setup/INSTALL_SUBMITTY_HELPER.sh

```diff
@@ -624,6 +624,12 @@ popd > /dev/null
 # COPY THE 1.0 Grading Website if not in worker mode
 if [ "${WORKER}" == 0 ]; then
     bash "${SUBMITTY_REPOSITORY}/.setup/install_submitty/install_site.sh"
+
+    if [ "${CI}" != true ]; then
+        echo -e "Checking for and fetching latest browscap.ini"
+        # fetch and convert browscap.ini to cache, may take some time on initial setup or if there's an update
+        ${SUBMITTY_INSTALL_DIR}/sbin/update_browscap.php "${PHP_USER}"
+    fi
```

Comment:
Let's move this block into `.setup/install_submitty/install_site.sh` to keep all the "site installation" code centralized.

Reply:
This has been moved to `.setup/install_submitty/install_site.sh`, but now every time `.setup/bin/code_watcher.py` detects a change, it checks for latest `browscap.ini`. Please see if that's right.
A possible solution can be to send an argument (or exporting a variable) to/for `.setup/install_submitty/install_site.sh` when invoked via `.setup/INSTALL_SUBMITTY_HELPER_SITE.sh` to disable the check for `browscap.ini` update.

Reply:
Now, `browscap` update is only checked if it's sent as an argument to `.setup/install_submitty/install_site.sh`. This is only done in `INSTALL_SUBMITTY_HELPER.sh`; this way, update check is avoided if `install_site.sh` runs separately at any other place.